lib/index.js

```diff
@@ -3,20 +3,17 @@
  */
 'use strict';
 
-const {config: {constants: bedrockConstants}} = require('bedrock');
+let {config: {constants: bedrockConstants}} = require('bedrock');
 const {jsonLdDocumentLoader} = require('bedrock-jsonld-document-loader');
 
 const contextModules = [
   'did-context',
 ];
 
 for(const module of contextModules) {
-  const {contexts, constants: contextConstants} = require(module);
-  for(const c in contextConstants) {
-    if(c.includes('URL')) {
-      const contextUrl = contextConstants[c];
-      bedrockConstants[c] = contextUrl;
-      jsonLdDocumentLoader.addStatic(contextUrl, contexts.get(contextUrl));
-    }
+  const {contexts, constants: moduleConstants} = require(module);
+  for(const [contextUrl, context] of contexts) {
+    jsonLdDocumentLoader.addStatic(contextUrl, context);
   }
+  bedrockConstants = {...bedrockConstants, ...moduleConstants};
```

Comment:
This won't work. `bedrockConstants` must be updated not a local copy just replaced.